payment-settler/test/worker.test.js

```diff
@@ -0,0 +1,375 @@
+import { describe, it, expect, beforeEach, afterEach, vi } from 'vitest'
+import { env } from 'cloudflare:test'
+import { withDataSet, randomId, getDaysAgo } from './test-helpers.js'
+import worker from '../bin/payment-settler.js'
+
+describe('payment settler scheduled handler', () => {
+  let mockPublicClient
+  let mockWalletClient
+  let mockAccount
+  let mockGetChainClient
+  let simulateContractCalls
+  let writeContractCalls
```

Comment:
Please add JSDoc comments to define types for these variables, so that we avoid `any` type.